external/loaders/loaders/batches/_batch.py

```diff
@@ -158,8 +161,7 @@ def batches_from_mapper(
 
     if timesteps is None:
         timesteps = list(data_mapping.keys())
-    shuffled_times = np.random.choice(timesteps, len(timesteps), replace=False).tolist()
-    batched_timesteps = list(partition_all(timesteps_per_batch, shuffled_times))
+    batched_timesteps = list(partition_all(timesteps_per_batch, timesteps))
```

Comment:
Reverted this to the original shuffling behavior, if `shuffle_samples=False` then the times are sorted in the list of transforms.

Reply:
Originally I suggested not having an option for this to keep the logic simple, since we have no use case that needs timestamp shuffling in the batches pipeline (always need to shuffle at train time, anyways).

I mostly think this should be consistent, so I'm fine with either of these:
- Add an option for shuffle_timesteps similar to shuffle_samples, because we want to be able to toggle shuffling options.
- Remove the keep_ordered_samples/shuffle_samples argument, because we don't need to be able to toggle shuffling options.

Reply:
> Add an option for shuffle_timesteps similar to shuffle_samples, because we want to be able to toggle shuffling options.

This was my suggestion above, so I agree that we should give a separate flag. 

With tf.data.Dataset we will still want to shuffle timesteps, since `.shuffle` only mixes things within a given buffer size. Mixing 3PM and 12PM of the same day together is not as much shuffling as mixing different days.